applications/RANSApplication/custom_processes/auxiliary_processes/rans_check_vector_bounds_process.cpp

```diff
@@ -113,36 +115,49 @@ void RansCheckVectorBoundsProcess::Execute()
         break;
     }
 
-    double min_value = 0.0;
-    double max_value = 0.0;
-    bool is_initialized = false;
+    double min_value = std::numeric_limits<double>::max();
+    double max_value = std::numeric_limits<double>::min();
```

Comment:
is it `min` or `lowest`? I am not sure, but I remember that I had an unplesant surprise some time ago

Reply:
you are correct, it should be lowest. thanks :)